test/spec/support/shell.rb

```diff
@@ -8,6 +19,16 @@ def run(cmd, opts = {})
     Kommando.run(cmd, opts)
   end
 
+  # @param [String] cmd
+  # @param [Hash] opts
+  # @raise [Error]
+  # @return [Kommando]
+  def run!(cmd, **opts)
+    k = run(cmd, **opts)
+    raise Error.new(cmd, k.code, k.out) if k.code != 0
+    return k
```

Comment:
doesn't need `return`

could even be:
```ruby
def run(!cmd, **opts)
  run(cmd, **opts).tap do |k|
    raise Error.new(cmd, k.code, k.out) unless k.code.zero?
  end
end
```

Reply:
I personally like the explicit `return` in this kind of case. The `tap` reads okay as well, should I change it to use that?

Reply:
Explicit return statement adds an extra method call compared to just `k`. but this is in test helper so it doesn't make any difference.